components/post_view/post_attachment_opengraph/post_attachment_opengraph.scss

```
@@ -0,0 +1,190 @@
+.truncate {
+    display: -webkit-box;
+    overflow: hidden;
+    -webkit-box-orient: vertical;
+    hyphens: auto;
+    -webkit-user-select: text;  /* Chrome all / Safari all */
+    -moz-user-select: text;     /* Firefox all */
+    -ms-user-select: text;      /* IE 10+ */
+    user-select: text;
+}
+
+.PostAttachmenOpenGraph {
```

I must've missed this earlier, but there's a typo here that's spread through the e2e tests and snapshot. Could you fix that?
```suggestion
.PostAttachmentOpenGraph {
```